README.md

```diff
@@ -28,9 +28,6 @@ Internally zombienet is a `javascript` library, designed to run on `Node.js` and
 backend `providers` to run the *nodes*, at this moment `kubernetes`, `podman` and `native` are
 supported.
 
-**Note:** Currently, it is only possible to use `podman` for Zombienet users on Linux machines. 
```

Comment:
This `Note` totally repeats `Note` [here](https://github.com/paritytech/zombienet/blob/a7a287f8237c2dcce1c3c65a52b4e1130a344461/README.md?plain=1#L41)

Reply:
applicable to all similar changes